sms_toolkit/messages/truncate.py

```diff
@@ -0,0 +1,59 @@
+from sms_toolkit import utils
+from sms_toolkit import constants
+from .normalization import normalize_to_unicode
+
+
+def truncate_message(message, max_length, is_for_mms=False):
+    """
+    Truncate the given SMS message string upto the provided length. If the SMS message length is less than
+    the length, or if the length is invalid (< 1), then the message is returned as is
+
+    Args:
+        message (basestring): The raw message string to profile.
+        max_length (int): The length upto which the message must be truncated. Must be greater than 0
+        is_for_mms (bool): Is this message for mms sending.
+
+    Returns:
+        (Text): The truncated message
+    """
+    if max_length < 1:
+        return normalize_to_unicode(message)
```

Comment:
**Question:** If length must be greater than 0, would we want to raise an exception here too?

Reply:
Yeah my thinking was raising an exception for incorrect max-length would be overkill. The only other place this method raises an exception is when it can't understand the encoding, which felt right. Lemme know if you feel strongly about this though